merlin-server/src/test/java/gov/nasa/jpl/aerie/merlin/server/mocks/StubMissionModelService.java

```diff
@@ -50,6 +51,8 @@ public final class StubMissionModelService implements MissionModelService {
       NONEXISTENT_ACTIVITY_TYPE,
       Map.of());
 
+  public static final Map<String, ValueSchema> RESOURCES;
```

Comment:
It would be more idiomatic to use the `Map.of` factory method here.

```suggestion
  public static final Map<String, ValueSchema> RESOURCES = Map.of(
    "mode", ValueSchema.ofVariant(List.of(
        new ValueSchema.Variant("Option1", "Option1"),
        new ValueSchema.Variant("Option2", "Option2")
    )),
    "state of charge", ValueSchema.REAL
  );
```